exonum-java-binding/core/rust/integration_tests/tests/service_runtime_bootstrap.rs

```diff
@@ -32,28 +35,18 @@ fn bootstrap() {
     };
 
     let runtime_config = RuntimeConfig {
+        artifacts_path: PathBuf::from("/tmp/"),
         // Pass log4j path to avoid error messages of mis-configuration
         log_config_path: log4j_path(),
         port: 6300,
         override_system_lib_path: None,
     };
 
-    let config = Config {
-        jvm_config,
-        runtime_config,
-    };
-
     let internal_config = InternalConfig {
         system_class_path: fakes_classpath(),
         system_lib_path: java_library_path(),
     };
 
-    let runtime = JavaServiceRuntime::new(config, internal_config);
-
-    let result = runtime
-        .get_executor()
-        .with_attached(|env| env.get_version());
-
-    assert!(result.is_ok());
-    assert!(i32::from(result.unwrap()) > 0);
+    let _runtime: Box<dyn Runtime> =
```

Comment:
This test shall perform some assertions on the created runtime/VM,
    shan't it?

Reply:
Actually yes. In previous versions of this test we used testkit to test the blockchain with our service proxy. I think we can do the same once Testkit will be ready.

For now this test can only spot if the runtime creation throws panic/returns error. Btw, the test is actually disabled and I never run it (it requires fakes which are broken right now).